s30157-MikolajMach/zajecia3/cwiczenie1/skrypt2.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+config_file="config.cfg"
+
+
+if [ -f "$config_file" ]; then
```

Comment:
A sprawdzasz czy masz uprawnienia o wykonania tego pliku? Czy ślepo zakładasz, że skoro będzie to na 100% będzie wykonywalny?